cmd/metrics-server/app/options/options.go

```diff
@@ -84,6 +86,7 @@ func NewOptions() *Options {
 		KubeletClient:  NewKubeletClientOptions(),
 
 		MetricResolution: 60 * time.Second,
+		MetricsAddress:   "0.0.0.0:8080",
```

Comment:
This is not a correct default, exposing unsecured endpoint should not be a default behavior. This endpoint should be optional.

Reply:
Is there a preferred way to define optional args, or should we use an empty string to indicate that this point is going to be disabled?